### ISSUE TITLE
explain output json text can not be parsed by python json.tool 

### ISSUE THREAD

**Issue by chenbaggio** [greenplum-db/gpdb#9410]
### Greenplum version or build
6.x_stable
### OS version and uname -a
centos
Linux computing-8c091891-master-6aadb510 3.10.0-957.21.3.el7.x86_64 #1 SMP Tue Jun 18 16:35:19 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
### Expected behavior

the output text can be parsed by python json.tool 

### Actual behavior

report errors:

`Expecting property name: line 181 column 16 (char 34043) ` etc.

### Step to reproduce the behavior

1. tpcds query 1 - attach the explain (verbose, format json) 

```
EXPLAIN (VERBOSE, FORMAT JSON)
with customer_total_return as
(select sr_customer_sk as ctr_customer_sk
,sr_store_sk as ctr_store_sk
,sum(SR_RETURN_AMT_INC_TAX) as ctr_total_return
from store_returns
,date_dim
where sr_returned_date_sk = d_date_sk
and d_year =1999
group by sr_customer_sk
,sr_store_sk)
 select  c_customer_id
from customer_total_return ctr1
,store
,customer
where ctr1.ctr_total_return > (select avg(ctr_total_return)*1.2
from customer_total_return ctr2
where ctr1.ctr_store_sk = ctr2.ctr_store_sk)
and s_store_sk = ctr1.ctr_store_sk
and s_state = 'SD'
and ctr1.ctr_customer_sk = c_customer_sk
order by c_customer_id
limit 100;
```
2. run the sql with the commad

```
psql -d gpadmin -f <query sql file> > <output file>

```
3.  handle the output

use vi editor to delete useless characters in the output text  and run command:

`cat <output file> | python -m json.tool`
report error:

`Expecting property name: line 41 column 16 (char 7575)`

**Comment by chenbaggio** [comment]
a hint - it seems somewhere miss the key "Plans" :

**Comment by hlinnaka** [comment]
Try using the 'psql -A -t' options. Otherwise there are headers and markers for line endings in the output. If it still fails, please attach the json file you got from that query.

**Comment by chenbaggio** [comment]
it is so pity! '-A -t ' options does not work, the output is 

[01.txt](https://github.com/greenplum-db/gpdb/files/4059425/01.txt)

**Comment by hlinnaka** [comment]
Great, thanks! Pretty straightforward bug, now that I know where to look. We failed to print the Plans (or corresponding tags for XML/YAML formats, I presume) for Sequence nodes. Opened PR #9423 to fix.

That query plan was really long, I hope this was the only error in it. If you can, please test your query with #9423 again to see if it works now. 

Thanks for the report!

**Comment by hlinnaka** [comment]
Fixed in 6X_STABLE and master, in commit af4e4cd9f398f2f0bcce6991d8b4e3a8d7a24bdc.
